domain/wallets/models/src/main/java/com/tangem/domain/wallets/models/UserWallet.kt

```diff
@@ -1,34 +1,37 @@
 package com.tangem.domain.wallets.models
 
+import com.tangem.domain.models.scan.CardDTO
 import com.tangem.domain.models.scan.ScanResponse
 
 /**
  * Represents user's wallet which stored in app persistence
- * @param name User wallet name
- * @param walletId User wallet [UserWalletId]
- * @param artworkUrl User wallet card artwork URL
- * @param cardsInWallet List of cards IDs assigned with this user's wallet
- * @param isMultiCurrency Indicates whether this user wallet can work with more than one currency
- * @param scanResponse [ScanResponse] of primary user's wallet card.
- * TODO: Replace with [com.tangem.domain.common.CardDTO]
- * @property cardId ID of user's wallet primary card
- * @property hasAccessCode Indicates if the user's wallet primary card has access code
- * @property isLocked Indicates if this primary card has no currency wallets
- * */
+ *
+ * @property name            User wallet name
+ * @property walletId        User wallet [UserWalletId]
+ * @property artworkUrl      User wallet card artwork URL
+ * @property cardsInWallet   List of cards IDs assigned with this user's wallet. It will be a non-empty list
+ *                              if card was backed up on current device.
```

Comment:
nit: Кажется стоит от обратного пойти) 
```suggestion
 * @property cardsInWallet   List of cards IDs assigned with this user's wallet. The list will be empty if the wallet has been backed up on another device.
```

Reply:
Если ты считаешь, что так понятнее будет